Dockerfile

```diff
@@ -0,0 +1,77 @@
+FROM ubuntu:xenial
```

Comment:
Can you add a comment at the top of this file saying that it's a fork of the other file and is for cloud build only? Maybe also add a comment to the other file saying it's for Travis?

Reply:
Done.